fleet-management/src/main/java/cz/fi/muni/pa165/entity/InspectionInterval.java

```diff
@@ -60,17 +61,20 @@ public void setDays(int days) {
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (o == null || (!(o instanceof InspectionInterval))) return false;
 
         InspectionInterval that = (InspectionInterval) o;
 
-        return id != null ? id.equals(that.id) : that.id == null;
+        if (days != that.getDays()) return false;
+        return name != null ? name.equals(that.getName()) : that.getName() == null;
 
     }
 
     @Override
     public int hashCode() {
-        return id != null ? id.hashCode() : 0;
+        int result = name != null ? name.hashCode() : 0;
+        result = 31 * result + days;
```

Comment:
Auto generate hashCode from idea, is fine, it just generate a "unique" number for specific entity, necessary in some collections.